source/features/quick-comment-edit.tsx

```diff
@@ -7,13 +7,13 @@ import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 
-function canEditDiscussion(): boolean {
-	return pageDetect.isDiscussion() && select.exists('[title^="You are a maintainer"], [title^="You are a collaborator"]');
+function canEditEveryComment(): boolean {
+	return select.exists('[aria-label^="You have been invited to collaborate"], [aria-label^="You are the owner"], [title^="You are a maintainer"], [title^="You are a collaborator"]') && select.exists('div.js-previewable-comment-form');
```

Comment:
Multiline, also no need for `^=` if we're matching the whole string, `=` is enough

```js
select.exists([
	'a',
	'b',
	'etc,
]) && select.exists('')
```

Reply:
I think the `You have been invited to collaborate` string has a suffix with the repo name, i.e., `You have been invited to collaborate on the ... repository`.